infra-templates/k8s/12/README.md

```diff
@@ -1,2 +1,6 @@
 ## Upgrade Kuberenetes Stack:  WARNING
 When upgrading to this version of Kubernetes, rolling back to the previous version is not supported.
+
+## Duration Format
+
+A duration string is a possibly signed sequence of decimal numbers, each with optional fraction and a unit suffix, such as "300ms", "-1.5h" or "2h45m". Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
```

Comment:
I find this is difficult to understand. What does a negative number mean? I think if we tell the user:
1. Valid formats with examples (expanding on whats already there maybe use the default 15m00s or whatever)
2. How to disable backups or set to a minimum.